config/settings.yml.example

```diff
@@ -56,3 +56,9 @@
 # Uncomment and modify if you want to change the log level
 # WARN, DEBUG, ERROR, FATAL, INFO, UNKNOWN
 #:log_level: ERROR
+
+# Log buffer size and extra buffer size (for errors)
+#:log_buffer: 2000
+#:log_buffer_errors: 1000
+# Log buffer threshold (debug messages are ignored by default)
+#:log_buffer_level: INFO
```

Comment:
Why do we need separate levels for the logger and the event buffer? The latter shouldn't filter any events, it's up to final consumers; otherwise it's possible to configure logger for debug, buffer for info, and in the end not see any debug-level messages in the log.